examples/bpi-monolith-test-suite/bpi/components/storage/agreement.ts

```diff
@@ -0,0 +1,75 @@
+import { Order } from "../../../domain-objects/order";
```

Comment:
just a thought ... why not have a module `domainObjects` and this module contains the modules from domain-object. hence you can address it by `domainObject.Order.getOrder` for example. @ognjenkurtic

Reply:
noted for future refactoring